Description: drop use of FT_CALLBACK_DEF() def
 From 2.10.3, Freetype disappeared the FT_CALLBACK_DEF() macro,
 which is what we used when defining our callbacks from Freetype.
 .
 No guidance forthcoming
 from the Freetype developer who made those changes,
 so change to explicitly declaring the callbacks file static.
 .
 Should fix the reported build failures.
Origin: upstream, https://git.ghostscript.com/?p=ghostpdl.git;h=41ef9a0
Author: Chris Liddell <chris.liddell@artifex.com>
Bug: https://bugs.ghostscript.com/show_bug.cgi?id=702985
Last-Update: 2020-12-23
---
This patch header follows DEP-3: https://dep.debian.net/deps/dep3/
--- a/base/fapi_ft.c
+++ b/base/fapi_ft.c
@@ -125,7 +125,7 @@
 delete_inc_int_info(gs_fapi_server * a_server,
                     FT_IncrementalRec * a_inc_int_info);
 
-FT_CALLBACK_DEF(void *)
+static void *
 FF_alloc(FT_Memory memory, long size)
 {
     gs_memory_t *mem = (gs_memory_t *) memory->user;
@@ -133,7 +133,7 @@
     return (gs_malloc(mem, size, 1, "FF_alloc"));
 }
 
-FT_CALLBACK_DEF(void *)
+static void *
     FF_realloc(FT_Memory memory, long cur_size, long new_size, void *block)
 {
     gs_memory_t *mem = (gs_memory_t *) memory->user;
@@ -153,7 +153,7 @@
     return (tmp);
 }
 
-FT_CALLBACK_DEF(void)
+static void
     FF_free(FT_Memory memory, void *block)
 {
     gs_memory_t *mem = (gs_memory_t *) memory->user;
